packages/oui-button/src/button.html

```diff
@@ -0,0 +1,20 @@
+<button
+  arial-label="{{$ctrl.text}}"
+  class="oui-button"
+  ng-attr-id="{{::$ctrl.id}}"
+  ng-attr-name="{{::$ctrl.name}}"
+  ng-attr-type="{{::$ctrl.type}}"
+  ng-class="{
```

Comment:
You could add one time binding, I don't think these values will change after first digest
 ng-class=":: {...

Reply:
Yes, done